test/code/plugins/out_oms_changetracking_file_systest.rb

```diff
@@ -0,0 +1,179 @@
+require 'fluent/test'
+require_relative ENV['BASE_DIR'] + '/source/ext/fluentd/test/helper'
+require_relative ENV['BASE_DIR'] + '/source/code/plugins/out_oms_changetracking_file'
+require_relative 'omstestlib'
+require_relative 'out_oms_systestbase'
+
+class TestableOutChangeTrackingFile < Fluent::OutChangeTrackingFile
```

Comment:
I thought of that, it will essentially mean storing the keys, which I did not want to do.
other way would be to integrate secret store, which is a lot more work again. so no, not this time.